src/controllers/usersController.js

```diff
@@ -0,0 +1,57 @@
+import models from '../database/models';
+import auth from '../helpers/authHelper';
+
+/**
+ * UserController class
+ */
+class UsersController {
+  /**
+   *
+   * @param {Object} req - server request
+   * @param {Object} res - server response
+   * @returns {Object} - custom response
+   */
+  static async signUp(req, res) {
```

Comment:
Why are you using a class here instead of a function? You're not extending any class neither will another class be inheriting this

Reply:
you could do
`
const functionA = () => {};
const functionB = () => {} ;

export {
  functionA,
  functionB
}
`

Reply:
Got it! I'm on it now.